packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -49,7 +49,7 @@ export const useButtonState = (draftState: ButtonState) => {
     }
   };
 
-  draftState['aria-disabled'] = draftState.disabled;
+  draftState['aria-disabled'] = draftState.disabled || draftState.disabledFocusable;
```

Comment:
disabledFocusable should add attribute `aria-disabled`, otherwise user has not information from screen reader that button is disabled